addons/respawn/functions/fnc_adjustTime.sqf

```diff
@@ -0,0 +1,24 @@
+#include "script_component.hpp"
+/*
+ * Author: 3Mydlo3
+ * Function raises adjustTimeLocal event on all clients.
+ *
+ * Arguments:
+ * 0: New respawn time delay <NUMBER>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [30] call afm_respawn_fnc_adjustTime
+ *
+ * Public: No
+ */
+
+params ["_newTime"];
+
+if (!isServer) exitWith {};
+
+[QGVAR(adjustTimeLocal), _this] call CBA_fnc_globalEvent;
+
+GVAR(oldTime) = _newTime;
```

Comment:
This will be not neccesary if we will adjust it in `onPlayerKilled`.

Reply:
This is server function, it won't have `onPlayerKilled` triggered. Assignment here just for consistency between server and clients.

Reply:
Dedicated does not need this variable, player server will have onPlayerKilled event

Reply:
So maybe i will remove it for dedicated entirely.